store/metrics.go

```diff
@@ -70,13 +79,29 @@ func (m *metrics) newHead(height uint64) {
 	})
 }
 
+func (m *metrics) newTail(height uint64) {
+	m.observe(context.Background(), func(context.Context) {
+		m.headHeight.Store(height)
+	})
+}
+
 func (m *metrics) observeHeight(_ context.Context, obs metric.Observer) error {
-	height := m.headHeight.Load()
-	if height > math.MaxInt64 {
-		return fmt.Errorf("height overflows int64: %d", height)
+	headHeight := m.headHeight.Load()
+	if headHeight > math.MaxInt64 {
```

Comment:
why do we do this again?

Also is it applicable to tail load?

Reply:
Linting issue, but generally a foot gun thay may backfire in a very long future